### DISCUSSION TITLE
Make it possible to change max width of action modal

### DISCUSSION THREAD

**Discussion by hivokas** [laravel/nova-issues#3950] [Ideas & Feature Requests] [answered]
Currently it's `xl` by default and there is no way to customize it. It would be great if it's possible, for example, to define `$maxWidth = '4xl'` inside the action class.

**Comment by goellner** [comment]
a really really dirty hack in a custom field: 

```
 mounted() {
        // dirty hack to make action modal wider
        this.$parent.contentClasses[0] = 'hp-max-w-6xl';
        this.$parent.$forceUpdate();
    },
 ```

**Comment by kusab85** [comment]
Yes, this hardcoded max width really suck:

![obraz](https://user-images.githubusercontent.com/53086618/188433837-3c5d25a0-be5f-4fa6-a363-6f133f1dfa5f.png)

**Comment by davidhemphill** [comment, marked as answer]
Happy to say we're adding configurable modal sizes to Nova in the next release. You'll be able to set the action modal size to sm, md, lg, xl, and 2xl. In addition, you'll be able to set the modal to be shown fullscreen (without any backdrop) to take advantage of your entire screen width.

**Reply by hivokas** [reply]
That's great, thanks!

**Reply by orlyapps** [reply]
whoooop <3
